pontoon/base/static/js/fluent_interface.js

```diff
@@ -106,7 +106,7 @@ var Pontoon = (function (my) {
           } else {
             cls = ' class="custom-attribute clearfix"';
             label = '<div class="wrapper">' +
-              '<input type="text" class="id" placeholder="enter-attribute-id" value="' + id + '">' +
+              '<input type="text" class="id" placeholder="enter-attribute-id" value="' + id + '" dir="' + Pontoon.locale.direction + '" data-script="' + Pontoon.locale.script + '"  lang="' + Pontoon.locale.code + '">' +
```

Comment:
This element is actually used for the attribute `id`, so no need to add the attributes here.

Reply:
Oops, reverted it.